code-submissions-clustering-plugin/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/cli/LoadRunner.kt

```diff
@@ -3,30 +3,47 @@ package org.jetbrains.research.code.submissions.clustering.cli
 import com.intellij.openapi.application.ApplicationStarter
 import com.intellij.openapi.diagnostic.Logger
 import com.xenomachina.argparser.ArgParser
+import com.xenomachina.argparser.default
 import org.jetbrains.kotlinx.dataframe.DataFrame
 import org.jetbrains.kotlinx.dataframe.io.readCSV
+import org.jetbrains.kotlinx.dataframe.io.writeCSV
 import org.jetbrains.research.code.submissions.clustering.load.unifiers.PyUnifier
 import org.jetbrains.research.code.submissions.clustering.util.loadGraph
+import org.jetbrains.research.code.submissions.clustering.util.toDataFrame
+import org.jetbrains.research.code.submissions.clustering.util.toProto
+import java.io.File
 import java.nio.file.Paths
 import kotlin.system.exitProcess
 
 object LoadRunner : ApplicationStarter {
     private val logger = Logger.getInstance(this::class.java)
-    private lateinit var inputFile: String
+    private var outputFilename: String? = null
+    private var csvFilename: String? = null
+    private lateinit var inputFilename: String
 
     override fun getCommandName(): String = "load"
 
     @Suppress("TooGenericExceptionCaught")
     override fun main(args: MutableList<String>) {
         try {
             ArgParser(args.drop(1).toTypedArray()).parseInto(::TransformationsRunnerArgs).run {
-                inputFile = Paths.get(input).toString()
+                inputFilename = Paths.get(input).toString()
+                outputFilename = output?.let { Paths.get(it).toString() }
+                csvFilename = outputCSV?.let { Paths.get(it).toString() }
```

Comment:
Let's ask the user about the output directory and store all representations in this directory with our own names